packages/hash/shared/src/blockMeta.ts

```diff
@@ -63,9 +57,7 @@ const toBlockConfig = (
 ): BlockConfig => {
   const defaultVariant: BlockVariant = {
     description: options.description ?? "",
-    displayName:
-      options.displayName ?? toDisplayName(options.name ?? "Unnamed Block"),
-    name: options.name,
+    name: options.displayName ?? options.name,
```

Comment:
`displayName` is deprecated in favor of `name` for `BlockVariant`s only, not for `BlockMetadata`, so I'm reverting this change for now.

Reply:
I think you misunderstood my comment. I had initially applied your suggestion, but it was not relevant because of reasons defined in my previous comment. Hence, I reverted the change that I had made and brought it back to the state it is presently at (which is the correct one).

Reply:
> By the way, the text is wrapping weirdly in the error box

Good catch, not sure where this is being introduced. Fixed here: https://github.com/hashintel/hash/pull/407/commits/979e99bfde300c15667ba51d1951bc41136a0a45.

@bw or @teenoh is this expected Mui behavior or should I create a task to debug this?